crypto/key_store_passphrase.go

```diff
@@ -112,40 +113,44 @@ func (ks keyStorePassphrase) GetKey(keyAddr []byte, auth string) (key *Key, err
 	return key, err
 }
 
-func (ks keyStorePassphrase) GetKeyAddresses() (addresses [][]byte, err error) {
+func (ks keyStorePassphrase) GetKeyAddresses() (addresses []common.Address, err error) {
 	return GetKeyAddresses(ks.keysDirPath)
 }
 
 func (ks keyStorePassphrase) StoreKey(key *Key, auth string) (err error) {
 	authArray := []byte(auth)
-	salt := randentropy.GetEntropyMixed(32)
+	salt := randentropy.GetEntropyCSPRNG(32)
```

Comment:
On Unix-like it uses dev/urandom, so will never block there.
Errors are checked: http://golang.org/pkg/crypto/rand/#Read
```
"On return, n == len(b) if and only if err == nil"
```

But let's double check exactly how it uses /dev/urandom and CryptGenRandom to be on the safe side.